aws/cloudformation-templates/base/cloudfront.yaml

```diff
@@ -98,6 +112,8 @@ Resources:
           TargetOriginId: S3
           ViewerProtocolPolicy: redirect-to-https
           CachePolicyId: !Ref UICachePolicy
+          OriginRequestPolicyId: !Ref UIOriginRequestPolicy
+          ResponseHeadersPolicyId: 60669652-455b-4ae9-85a4-c4c02393f86c
```

Comment:
Where does this id come from? Can it be parameterized?

Reply:
This is the id for the [SimpleCORS](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/using-managed-response-headers-policies.html#managed-response-headers-policies-cors) AWS Managed response header policies for cloudfront. should be able to be parametrized. I'll see if I can make the change, if I can't will ask for help

Reply:
No need if it's standard, but I'd probably add a comment so people don't wonder in the future